source/includes/pluggable-scm/_02-create.md.erb

```diff
@@ -0,0 +1,89 @@
+## Create a SCM object
+
+```shell
+$   curl 'http://ci.example.com/go/api/admin/scms' \
+  -u 'username:password' \
+  -H 'Accept:application/vnd.go.cd.v1+json' \
+  -H 'Content-Type:application/json' \
+  -X POST -d '{
+  "id": "scm-id",
+  "name": "foobar",
+  "auto_update": true,
+  "plugin_configuration": {
+    "id": "github.pr",
+    "version": "1"
+  },
+  "configuration": [
+    {
+      "key": "url",
+      "value": "git@github.com:sample/example.git"
+    },
+    {
+      "key": "username",
+      "value": "admin",
+      "secure": true
+    },
+    {
+      "key": "password",
+      "encrypted_value": "aSdiFgRRZ6A=",
+      "secure": true
+    }
+  ]
+ }'
+```
+
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.go.cd.v1+json; charset=utf-8
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "http://ci.example.com/go/api/admin/scms/foobar"
+    },
+    "doc": {
+      "href": "http://api.go.cd/#scms"
```

Comment:
You had asked me to change it from pluggable-scms to scms in an earlier conversation :)